src/babel.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+var _ = require('underscore-plus')
+var crypto = require('crypto')
+var path = require('path')
+var defaultOptions = require('../static/babelrc.json')
```

Comment:
After thinking about this more, it probably isn't worth the effort/complexity, feel free to disregard.